paf-mvp-client-express/src/client-node.ts

```diff
@@ -142,7 +142,6 @@ export class ClientNode extends Node {
     // *****************************************************************************************************************
     this.setEndpointConfig('GET', redirectProxyEndpoints.read, {
       endPointName: 'RedirectRead',
-      isRedirect: true,
```

Comment:
This field is used in error handler to redirect to returnUrl when something goes wrong. It is to be used by operatorNode only as the clientNode is only responsible for providing the url.

Reply:
Correct. We should refactor this to rename it, then. It will be misleading that we don't use it here.
I suggest we rename it to `redirectResponse` or something like this, and add a comment in the client node class to justify we don't use (basically what you wrote here).